language_detector/main.py

```diff
@@ -2,8 +2,52 @@
 
 from .languages import LANGUAGES
 
+def load_languages(language_list=LANGUAGES):
+    # Input: A list of dicts where within each dict in the list, the value for the key 
+    #   'name' is the language and the value key 'common words' is a list of common words. 
+    # Returns:  A dict with the key being the names of the lanauges and the values 
+    #   a int initialized to zero.
+    
+    result = {}
+    for lang_dict in language_list:
+        result[lang_dict.get('name')] = 0
+    return result
+
```

Comment:
While I think making another function is also completely fine, I once read that if you do something more than 3 times, you should make it a function instead of hard coding it each time.

From what I see, you only do load_languages once, and you never change anything in it, you always load your language files, nothing changes there either, so you really wouldn't need to make it a function. 

Not a big deal obviously, but I think it's over complicating it a bit.